frontend/src/components/Layout.tsx

```diff
@@ -18,22 +18,24 @@ const Content = styled.main`
 `;
 
 interface LayoutProps {
+  shouldAddPadding: boolean;
   hideBar: boolean;
   children: JSX.Element | string;
 }
 
 // Main layout of the project that includes both header and navbar
-const Layout = ({ hideBar = false, children } : LayoutProps) : ReactElement => {
+const Layout = ({ hideBar = false, shouldAddPadding = true, children } : LayoutProps) : ReactElement => {
   return (
     <Wrapper>
       <Nav showsBar={!hideBar}/>
-      <Content shouldAddPadding={!hideBar}>{children}</Content>
+      <Content shouldAddPadding={!hideBar && shouldAddPadding}>{children}</Content>
```

Comment:
Minor thing tho, and I am super sure if this is the file lol. The texts on the landing page are kinda not at the center. Is it possible to adjust it to be like about the center on the right panel :>